src/libs/Navigation/extrapolateStateFromParams.ts

```diff
@@ -0,0 +1,30 @@
+import type {InitialState} from '@react-navigation/native';
+
+/**
+ * It's possible that information from params about which screen we want to open is not in the state yet.
+ * In that case this function will state with information from params.
```

Comment:
I'm not sure what this sentence is saying. Is this right?
```suggestion
 * In that case this function will return the state with information from params.
```